dist/templates/widget-cookie-declaration/consent-cookie-declaration.css

```diff
@@ -92,42 +89,25 @@
 }
 
 .cookies-list__cookies-list-table {
-  table-layout: fixed;
-  width: 100%;
-  border-collapse: collapse;
-  border-spacing: 0;
-  margin: 0 0 18px 0;
-  padding: 0;
-  color: #757575;
-  font-size: 13px;
-  font: inherit;
-  vertical-align: baseline;
-  border-left: none;
-  border-right: none;
+  border-top: 1px solid #e0e0e0;
+  border-bottom: 1px solid #e0e0e0;
 }
 
-.cookies-list-table__header {
-  font-weight: bold;
-  border-bottom: 1px solid #e0e0e0;
-  text-align: left;
-  padding: 4px;
-  font-size: 12px;
+.coi-consent-banner__cookie-details {
   color: #757575;
```

Comment:
We should try to keep most popular convention with ordering CSS properties - color should be one of the last, check out: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/